### PR TITLE
Fix invalid toolConfigurationNotification property `suppressions` in …

### PR DESCRIPTION
**Pull request by yongyan-gh**
# Description
With latest version of ESLint SARIF formatter, GitHub workflow may fail to upload SARIF report to GitHub, if the results contains any tool configuration notifications due to an additional property 'suppressions' added to notification which violates the definition of JSON schema.

Errors from workflow:
```
[build](https://github.com/yongyan-gh/vscode-custom-data/runs/5397379598?check_suite_focus=true)
Unable to upload "eslint.sarif" as it is not valid SARIF:
- instance.runs[0].invocations[0].toolConfigurationNotifications[0] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[1] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[2] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[3] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[4] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[5] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[6] additionalProperty "suppressions" exists in instance when not allowed
```

# Fix
Do not generate `suppressions` properties for toolConfigurationNotifications.
Create a workflow to run local ESLint SARIF formatter to verify the SARIF file is valid.

### CODE REVIEW

File: .github/workflows/eslintsarif-test.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        uses: actions/upload-artifact@v2
+        with:
+          name: eslint-results
+          path: eslint-results.sarif
+      
+      - name: Validate sarif file
+        run: npx @microsoft/sarif-multitool validate eslint-results.sarif
+
+      - name: Upload SARIF file
+        uses: github/codeql-action/upload-sarif@v1
```
Comment (eddynaka) [inline .github/workflows/eslintsarif-test.yml:44]: >github/codeql-action/upload-sarif [](http://example.com/codeflow?start=14&length=33)

enable the option to wait for it to be processed.

**Comment by eddynaka** [comment]
Update the version of the extension as well.

**Review by michaelcfanning — APPROVED** [review]
:shipit:

**Review by eddynaka — APPROVED** [review]
:shipit:

**Pull request merged**